chapters/05-summary.Rmd

```diff
@@ -1,3 +1,23 @@
 # Summary {#summary}
 
-In summary we can conclude that...
\ No newline at end of file
+The benchmarking has shown that the new implementation can outperform the de-facto state of the art library 'KDEpy' in terms of runtime.
+
+In terms of accuracy it is slightly better in most cases and in general comparable to KDEpy.
+
+Additionally it has the benefit of allowing any distribution of the loc-scale family as kernel implemented in TensorFlow Probability[@googleTFP]. At the moment this includes twelve kernels, namely `Cauchy`, `DoublesidedMaxwell`, `Gumbel`, `Laplace`, `LogLogistic`, `LogNormal`, `Logistic`, `LogitNormal`, `Moyal`, `Normal`, `PoissonLogNormalQuadratureCompound`, `SinhArcsinh`.
+
+The proposed implementation restricts itself to the one-dimensional case. So far only KDEpy and Statsmodels implement a multidimensional KDE.
```

Comment:
[5] why to 1d? Don't forget the story: we want the 1d high performance case here AND the multidim is different (explain this somewhere?)